### ISSUE TITLE
consider removing bettercombat compatibility for base hephaestus

### ISSUE THREAD

**Issue by ceidways** [tiigerbones/hephaestusexpansion#2]
had some issues trying to remove bettercombat animations for base tinkers pickaxe, pickadze and mattock (had issues with attacking enemies when trying to mine). 
Was using a datapack to add bettercombat compatability for these things, and was confused when deleting the .json did nothing. After a lot of troubleshooting I realised that you add bettercombat support for base tinkers as well as the stuff you add. After I realised this it was relatively easy to just delete the folder in the .jar, though mildly inconvenient.

Adding compatibility for base stuff as well might cause other people confusion as well, I can imagine that there would be other scenarios where this would be frustrating, and if people want it, it isn't too much effort to get a datapack specifically for this.

Hope you'll consider, great little mod btw.

**Comment by tiigerbones** [comment]
Noted!
I'll make sure to make these changes on the next update!

**Comment by DurekuRyuu** [comment]
Any chance you could point me at the source of that datapack @ceidways ? I'd like to attempt a patch to the Hephaestus Expansion weapons for my server and would like something to reference as this would be my first datapack and I'm intimidated.

**Comment by ceidways** [comment]
I referenced these two and kinda combined them/changed a few things. It's been a while so I cant remember exactly what. Think it was something about better combat attack being annoying when trying to mine with pickaxes or other tools, and no one in my group used them as weapons so I simply removed the compatibility .json for them

https://www.curseforge.com/minecraft/mc-mods/bettercombat-forge-tconstruct
https://www.curseforge.com/minecraft/texture-packs/tinkers-better-combat

Found my one, don't mind the filename lol.
[sexytinkersbettercombat.zip](https://github.com/user-attachments/files/18838693/sexytinkersbettercombat.zip)

Better combat has documentation for this, though it is a bit wordy for what it is. 
I've linked the "Basics" section of their documentation, for just the simplest part of what you'd need to understand **probably read from "Let's see an example..." to the end of the "Using a preset" section**.
https://github.com/ZsoltMolnarrr/BetterCombat?tab=readme-ov-file#basics

If you want to do custom animations and stuff rather than using the existing presets, then it seems that'll require a bit more effort and reading, but if you decide to put in the effort then I wish you luck!

**Comment by tiigerbones** [comment]
1e5b602
